files/livefyre/info.ini

```diff
@@ -0,0 +1,5 @@
+author = "LiveFyre"
+github = "https://github.com/itseasy21/jsdelivr"
```

Comment:
I got it from : http://zor.livefyre.com/wjs/v3.0/javascripts/livefyre.js
It is hosted on livefyre servers, which is the comment system that i am using, but they dont have a cdn and neither their JS is available on any public CDN.